### ISSUE TITLE
access: may-start-trybots

### ISSUE THREAD

**Issue by abner-chenc** [golang/go#66558]
Hi,

I'd like to request the "may-start-trybots" access (https://go-review.googlesource.com/#/admin/groups/1030,members) for myself .

- gerrit account: chenguoqi@loongson.cn
- github account: [abner-chenc](https://github.com/abner-chenc)

I'm already part of the @golang/loong64 team for a while, and I'll be helping gardening this port in the UTC+8 timezone for the foreseeable future. It's increasingly inconvenient to have to ping other people for TryBot runs as the number of outstanding loong64 CLs keeps growing.

See https://go.dev/wiki/GerritAccess

Thanks.

**Comment by dmitshur** [comment]
Hi abner chenc,

Thanks for your contributions and helping maintain the linux/loong64 port. We've added you to the `project-golang-may-start-trybots` group, so you should be able to trigger [trybots](https://go.dev/wiki/LUCI#trybots) now.

As noted at https://go.dev/wiki/GerritAccess#running-trybots-may-start-trybots, please be sure to at least skim CLs for anything malicious before starting a trybot run.

Please let us know if you have any questions or issues. Thanks again.

**Comment by abner-chenc** [comment]
I can't select tryjobs, do I need additional permissions?  @dmitshur

**Comment by abner-chenc** [comment]
> The gerrit account `chenguoqi@loongson.cn` is still present in the `project-golang-may-start-trybots` group, so you should be able to. Are you logged in with that account and following the directions described at https://go.dev/wiki/LUCI#trybots?

Yes,
When I click the CHOOSE TYRJOBS button to select one of the builders, such as gotip linux-loong64, and click the Add button, it fails, and an error message pops up saying "Failed to create builds." Please try again See console logs, too There is no other useful information left.

**Comment by dmitshur** [comment]
To confirm, you are able to leave a Commit-Queue+1 vote using your account, is that right? I see [here](https://go-review.googlesource.com/c/go/+/544435/1#message-07e9ce8108aeb839e2e9be291985b79117b0351d) that is so.

In that case, the may-start-trybots access is successfully granted to your Gerrit account, but this appears to be something else going wrong when using the Gerrit UI. Please file a new issue with a prefix "gerrit: " so we can continue the investigation there.
